doc/src/site/sphinx/migration_guide.rst

```diff
@@ -51,22 +51,22 @@ probabilities, contributions and so on.
 Removal of Deprecated Methods and Classes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- ``getColsampleBytree and setColsampleBytree`` methods are removed on our algorithm API. Please use
-  the new ``getColSampleByTree`` and ``setColSampleByTree``
+- ``getColsampleBytree`` and ``setColsampleBytree`` methods are removed from the XGBoost API. Please use
+  the new ``getColSampleByTree`` and ``setColSampleByTree``.
 
-- Remove deprecated option ``spark.ext.h2o.external.cluster.num.h2o.nodes`` and corresponding setters.
-  Please use ``spark.ext.h2o.external.cluster.size`` or corresponding setter ``setClusterSize``
+- Removal of deprecated option ``spark.ext.h2o.external.cluster.num.h2o.nodes`` and corresponding setters.
+  Please use ``spark.ext.h2o.external.cluster.size`` or the corresponding setter ``setClusterSize``.
 
-- Deprecated Algorithm classes in package ``org.apache.spark.h2o.ml.algos`` have been removed. Please
-  use the classes from the package ``ai.h2o.sparkling.ml.algos``. Their API remains the same. This is the
-  beginning of moving Sparkling Water classes to our distinct package.
+- Removal of of deprecated algorithm classes in package ``org.apache.spark.h2o.ml.algos``. Please
+  use the classes from the package ``ai.h2o.sparkling.ml.algos``. Their API remains the same as before. This is the
+  beginning of moving Sparkling Water classes to our distinct package ``ai.h2o.sparkling``
 
-- Remove deprecated option ``spark.ext.h2o.external.read.confirmation.timeout`` and related setters.
-  This option is removed without replacement as it is no longer needed.
+- Removal of deprecated option ``spark.ext.h2o.external.read.confirmation.timeout`` and related setters.
+  This option is removed without a replacement as it is no longer needed.
 
-- Remove deprecated parameter ``SelectBestModelDecreasing`` on our Grid Search API. Related getters and setters
-  have been also removed. This method is removed without replacement as we now internally correctly sort
-  the models according to what is best for the specified metric.
+- Removal of deprecated parameter ``SelectBestModelDecreasing`` on the Grid Search API. Related getters and setters
+  have been also removed. This method is removed without replacement as we now internally sort
+  the models with the ordering meaningful for the specified sort metric.
```

Comment:
nit: maybe `meaningful to` would be better

Reply:
Oki, will fix in the following small change, thanks!